artifactory/utils/container/image.go

```diff
@@ -15,21 +19,36 @@ func (image *Image) Tag() string {
 }
 
 // Get image relative path in Artifactory.
-func (image *Image) Path() string {
+func (image *Image) Path() (string, error) {
+	if err := image.validateTag(); err != nil {
+		return "", err
+	}
 	indexOfFirstSlash := strings.Index(image.tag, "/")
 	indexOfLastColon := strings.LastIndex(image.tag, ":")
 	if indexOfLastColon < 0 || indexOfLastColon < indexOfFirstSlash {
-		return path.Join(image.tag[indexOfFirstSlash:], "latest")
+		log.Info("The image '%s' is missing tag. using the 'latest' tag.")
```

Comment:
The image '%s' is missing tag. using the 'latest' tag.
-->
The image '%s' is does not include tag. Using the 'latest' tag.